File: DC_CS422_Lab4/Week4Lab-Android/app/src/main/java/cs/mad/week4lab/entities/FlashcardSet.kt
package cs.mad.week4lab.entities

data class FlashcardSet(val title: String)

fun getFlashcardSets(): List<FlashcardSet> {
    val sets = mutableListOf<FlashcardSet>()
    for (i in 0..9) {
        sets.add(FlashcardSet("Set $i"))
    }
    return sets
}

File: DC_CS422_Lab5/Week5Lab-iOS/Week5Lab-iOS/Week5Lab/entities/FlashcardSet.swift
//
//  FlashcardSet.swift
//
//  Created by Andrew Taylor on 1/22/23.
//

import Foundation

class FlashcardSet {
    var title: String
    
    init(title: String) {
        self.title = title
    }
}

func getFlashcardSets() -> [FlashcardSet]
{
    var sets = [FlashcardSet]()
    for i in 1...10
    {
        sets.append(FlashcardSet(title: "Title \(i)"))
    }
    return sets
}
